### PR TITLE
[Clang-tidy header][18/N] Enable clang-tidy on headers in torch/csrc/cuda

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Comment by colesbury** [comment]
@maleft would you please review this?

File: torch/csrc/cuda/Module.cpp [hunk 1]
```diff
@@ -189,7 +189,8 @@ PyObject* THCPModule_getCurrentStream_wrap(
   TORCH_CHECK(
       THPUtils_checkLong(device_index), "invalid argument to getCurrentStream");
   int64_t device = THPUtils_unpackLong(device_index);
```
Comment (Skylion007) [inline torch/csrc/cuda/Module.cpp:191]: We should be consistent with how we are unpacking the long (or int)?

Comment (cyyever) [reply torch/csrc/cuda/Module.cpp:191]: Let's use long

**Comment by cyyever** [comment]
@Skylion007 The device index casting was move to other PRs.

**Review by albanD — APPROVED** [review]
Thanks!

FYI @guangyey more fixes that you're not getting...

**Comment by guangyey** [comment]
> Thanks!
> 
> FYI @guangyey more fixes that you're not getting...

Thanks very much, I will submit a PR to follow this.

**Pull request closed without merging**